_episodes/01-introduction.md

```diff
@@ -13,12 +13,47 @@ keypoints:
 - "This training aims to build a shared understanding and facilitate the integration of computational reproducibility in data science."
 ---
 
-# Data Science for Biomedical Scientists Project
+# Managing Open and Reproducible Computational Projects
+
+Over the last decade, several tools, methods and training resources have been developed for early career researchers to learn about and apply data science skills in biomedicine.
+This is often referred to as biomedical data science, with the following definition. 
+
+
+>Biosciences and biomedical researchers regularly combine mathematics and computational methods to interpret experimental data. The term “data science” describes expertise associated with taking (usually large) data sets and annotating, cleaning, organizing, storing, and analyzing them for the purposes of extracting knowledge. [...]
+>The terms “biomedical data science” and “biomedical data scientist” [...] connote activities associated with the creation and application of methods to new and large sources of biological and medical data aimed at converting them into useful information and knowledge. 
+>They also connote technical activities that are data-intensive and require special skills in managing the large, noisy, and complex data typical of biology and medicine. 
+>They may also imply the application of these technologies in domains where their collaborators previously have not needed data-intensive computational approaches. 
+>
+> -- **Russ B. Altman and Michael Levitt** (2018). [Annual Review of Biomedical Data Science](https://www.annualreviews.org/doi/full/10.1146/annurev-bd-01-041718-100001)
+
+
+In contrast to the definition above (and as will be explained in the next chapter), we think research which is not data intensive would also gain in applying data science principles.
+However, to ensure that data science approaches are appropriately applied in domain research, such as in biosciences, there is a need to also engage and educate scientific group leaders and researchers in project leadership roles on best practices.
```

Comment:
```suggestion
This training material gives a particular emphasis on building project leadershiping skills, as we think it will be critical for the adoption of new approaches in research.
This part of the training is aimed both at 
 scientific group leaders and researchers (that is future leaders).
```

Reply:
@malvikasharan is that what you meant here ?